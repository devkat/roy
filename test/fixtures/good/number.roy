// Test some large exponents
console.log 1e32
console.log 1E32
console.log 1e+32
console.log 1E+32
console.log 1e-32
console.log 1E-32

// Test some small exponents
console.log 1E+3
console.log 1E-3
