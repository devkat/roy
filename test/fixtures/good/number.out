1e+32
1e+32
1e+32
1e+32
1e-32
1e-32
1000
0.001
